packages/netlify-cms-core/src/components/Editor/EditorToolbar.js

```diff
@@ -582,19 +584,14 @@ export class EditorToolbar extends React.Component {
         t('editor.editorToolbar.deleteUnpublishedEntry')) ||
       (!hasUnpublishedChanges && !isModification && t('editor.editorToolbar.deletePublishedEntry'));
 
-    return [
-      <SaveButton key="save-button" onClick={() => hasChanged && onPersist()}>
-        {isPersisting ? t('editor.editorToolbar.saving') : t('editor.editorToolbar.save')}
-      </SaveButton>,
-      (!showDelete || useOpenAuthoring) && !hasUnpublishedChanges && !isModification ? null : (
-        <DeleteButton
-          key="delete-button"
-          onClick={hasUnpublishedChanges ? onDeleteUnpublishedChanges : onDelete}
-        >
-          {isDeleting ? t('editor.editorToolbar.deleting') : deleteLabel}
-        </DeleteButton>
-      ),
-    ];
+    return !showDelete && !hasUnpublishedChanges && !isModification ? null : (
```

Comment:
Doesn't this change breaks https://github.com/netlify/netlify-cms/pull/5568?

Reply:
I have fixed this in the latest commit